packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -99,6 +103,13 @@ const POISubmission = ({
                     default:
                         break;
                 }
+            } else if (
+                mismatch_status &&
+                ![(idv_error_statuses.poi_expired, idv_error_statuses.poi_failed)].includes(mismatch_status) &&
```

Comment:
Do we need these additional parentheses here?

Reply:
Will remove it